system/index.js

```diff
@@ -49,9 +52,9 @@ module.exports = async ({headers, payload}) => {
 
   if (DEBUG) {
     console.log('==Middleware Function(s) End==');
-    console.log(`USER FUNCTION RECEIVED PAYLOAD: ${payload}`);
+    console.log(`USER FUNCTION RECEIVED: ${res}`);
   }
-  const result = await userFn(payload);
+  const result = await userFn(...res);
```

Comment:
If there is no middleware or the middleware returns nothing it looks like the `userFn` no longer get the payload?

Reply:
Yeah, I need to fix that.  Good catch 👍